lambda/voting/backend/server.js

```diff
@@ -153,3 +193,32 @@ async function invokeLambdaFunction(requestBody) {
   console.log("\n");
   return resultJson;
 }
+
+// Utility function to read from Lambda Function through MetaKeep Lambda Read API.
+
+async function readLambdaFunction(requestBody) {
+  const url = "https://api.metakeep.xyz/v2/app/lambda/read/";
+  const headers = {
+    "Content-Type": "application/json",
+    Accept: "application/json",
+    "x-api-key": process.env.API_KEY,
+    "Idempotency-Key": "Idempotency-Key" + Math.random().toString(),
+  };
+
+  const options = {
+    method: "POST",
+    headers: headers,
+    body: JSON.stringify(requestBody),
+  };
+  const result = await fetch(url, options);
+  const resultJson = await result.json();
+  console.log("Lambda Invocation response: ");
+  console.log(resultJson);
+  if (!result.ok) {
```

Comment:
need to check result status too since we can have FUNCTION_FAILURE

Reply:
done